.readthedocs.yaml

```diff
@@ -0,0 +1,38 @@
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
```

Comment:
after this pr, we need to have registered in readthedoc. They will constantly fetch the repo or get the notification by some API hook. We do not need to do anything on our CI.